src/main/java/com/bobocode/mapper/NoteRepresentationMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.bobocode.mapper;
+
+import com.bobocode.controller.UserController;
+import com.bobocode.dto.Note;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.hateoas.server.RepresentationModelAssembler;
+import org.springframework.stereotype.Component;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@Component
```

Comment:
is this class required to be a bean?

Reply:
Yep, I inject this bean in the controller. Why do you think it is not required?

Reply:
no-no, I just clarified. I've missed that it's injected in other bean)